public/js/pimcore/object/tags/abstractRelations.js

```diff
@@ -19,6 +19,34 @@ pimcore.object.tags.abstractRelations = Class.create(pimcore.object.tags.abstrac
 
     getFilterEditToolbarItems: function () {
         return [
+            {
+                iconCls: "pimcore_icon_clear_filters",
+                itemId: "clearFilters",
+                hidden: true,
+                text: t("clear_filters"),
+                handler: function (button) {
+                    this.component.filters.clearFilters();
+                    this.component.getStore().clearFilter();
+
+                    let columns = this.component.getColumns();
+                    for (let i = 0; i < columns.length; i++) {
+                        if(columns[i].filter.menu) {
```

Comment:
```suggestion
                        if(columns[i].filter?.menu?.items) {
```
There are columns that have no filter, eg. the sorting/open/delete ones

<img width="480" height="300" alt="image" src="https://github.com/user-attachments/assets/e41210b8-590e-47f7-944b-5d869d0a8fa2" />